### PR TITLE
Web Api

### PR DESCRIPTION
**Pull request by LondonAtlas**

### CODE REVIEW

File: PetAdoption-iOS/PetAdoptionTransportKit/Behavior.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+//  Behavior.swift
+//  PetAdoption-iOS
+//
+//  Created by Tom Marks on 21/08/2016.
+//  Copyright © 2016 Code For Orlando. All rights reserved.
+//
+
+import Foundation
+
+public protocol Behaviour {
```
Comment (WERUreo) [inline PetAdoption-iOS/PetAdoptionTransportKit/Behavior.swift]: This is 'murica pal! Behaviour.... :P

File: PetAdoption-iOS/PetAdoption-iOS.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -47,6 +47,33 @@
...
+		D4F2A7321D6A081F00E92647 /* PetAdoptionTransportKit.h in Headers */ = {isa = PBXBuildFile; fileRef = D4F2A7311D6A081F00E92647 /* PetAdoptionTransportKit.h */; settings = {ATTRIBUTES = (Public, ); }; };
+		D4F2A7391D6A081F00E92647 /* PetAdoptionTransportKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D4F2A72F1D6A081F00E92647 /* PetAdoptionTransportKit.framework */; };
+		D4F2A7401D6A081F00E92647 /* PetAdoptionTransportKitTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = D4F2A73F1D6A081F00E92647 /* PetAdoptionTransportKitTests.swift */; };
+		D4F2A7441D6A081F00E92647 /* PetAdoptionTransportKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D4F2A72F1D6A081F00E92647 /* PetAdoptionTransportKit.framework */; };
+		D4F2A7451D6A081F00E92647 /* PetAdoptionTransportKit.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = D4F2A72F1D6A081F00E92647 /* PetAdoptionTransportKit.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
+		D4F2A74E1D6A084B00E92647 /* PTKRequestManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = D4F2A74D1D6A084B00E92647 /* PTKRequestManager.swift */; };
```
Comment (WERUreo) [inline PetAdoption-iOS/PetAdoption-iOS.xcodeproj/project.pbxproj]: This file is coming up missing in the version I have...

File: PetAdoption-iOS/PetAdoptionTransportKit/Behavior.swift (hunk 1 above)
Comment (LondonAtlas) [inline PetAdoption-iOS/PetAdoptionTransportKit/Behavior.swift]: shit, good point.

File: PetAdoption-iOS/PetAdoption-iOS.xcodeproj/project.pbxproj (hunk 2 above)
Comment (WERUreo) [inline PetAdoption-iOS/PetAdoption-iOS.xcodeproj/project.pbxproj]: PTKRequestManager.swift, to be specific

**Comment by WERUreo** [comment]
Please see PR #35

**Pull request closed without merging**